sunpy/coordinates/frames.py

```diff
@@ -137,7 +137,45 @@ def __str__(self):
 
 
 @add_common_docstring(**_frame_parameters())
```

Comment:
```suggestion
```
The docstring for this base class isn't making use of the `add_common_docstring()` functionality.